terraform_apply.sh

```diff
@@ -0,0 +1,138 @@
+    -target=aws_security_group_rule.allow_nfs_inbound \
+    -target=aws_security_group_rule.allow_nfs_outbound \
+    -target=aws_security_group_rule.egress_from_cloudwatch_agent_endpoint_client_to_cloudwatch_agent_endpoint_via_https \
+    -target=aws_security_group_rule.egress_from_ec2_endpoint_client_to_ec2_endpoint_via_https \
+    -target=aws_security_group_rule.egress_from_ssm_agent_endpoint_client_to_ssm_agent_endpoint_via_https \
+    -target=aws_security_group_rule.egress_from_ssm_endpoint_client_to_ssm_endpoint_via_https \
+    -target=aws_security_group_rule.egress_from_sts_endpoint_client_to_sts_endpoint_via_https \
+    -target=aws_security_group_rule.egress_to_dynamodb_endpoint_via_https \
+    -target=aws_security_group_rule.egress_to_s3_endpoint_via_https \
+    -target=aws_security_group_rule.ingress_from_cloudwatch_agent_endpoint_client_to_cloudwatch_agent_endpoint_via_https \
+    -target=aws_security_group_rule.ingress_from_ec2_endpoint_client_to_ec2_endpoint_via_https \
+    -target=aws_security_group_rule.ingress_from_ssm_agent_endpoint_client_to_ssm_agent_endpoint_via_https \
+    -target=aws_security_group_rule.ingress_from_ssm_endpoint_client_to_ssm_endpoint_via_https \
+    -target=aws_security_group_rule.ingress_from_sts_endpoint_client_to_sts_endpoint_via_https \
+    -target=aws_security_group_rule.ingress_from_teamserver_to_gophish_via_smtp \
+    -target=aws_security_group_rule.scanner_egress_to_anywhere_via_any_port \
+    -target=aws_security_group_rule.smb_client_egress_to_smb_server \
+    -target=aws_security_group_rule.smb_server_ingress_from_smb_client \
+    -target=aws_vpc_endpoint.dynamodb \
+    -target=aws_vpc_endpoint.ec2 \
+    -target=aws_vpc_endpoint.ec2messages \
+    -target=aws_vpc_endpoint.kms \
+    -target=aws_vpc_endpoint.logs \
+    -target=aws_vpc_endpoint.monitoring \
+    -target=aws_vpc_endpoint.s3 \
+    -target=aws_vpc_endpoint.ssm \
+    -target=aws_vpc_endpoint.ssmmessages \
+    -target=aws_vpc_endpoint.sts \
+    -target=aws_vpc_endpoint_route_table_association.s3_operations \
+    -target=aws_vpc_endpoint_route_table_association.s3_private \
+    -target=aws_vpc_endpoint_subnet_association.ec2 \
+    -target=aws_vpc_endpoint_subnet_association.ec2messages \
+    -target=aws_vpc_endpoint_subnet_association.kms \
+    -target=aws_vpc_endpoint_subnet_association.logs \
+    -target=aws_vpc_endpoint_subnet_association.monitoring \
+    -target=aws_vpc_endpoint_subnet_association.ssm \
+    -target=aws_vpc_endpoint_subnet_association.ssmmessages \
+    -target=aws_vpc_endpoint_subnet_association.sts \
+    -target=null_resource.break_association_with_default_route_table \
+    -target=module.email_sending_domain_certreadrole \
+    -target=module.guacamole_certreadrole \
+    -target=module.read_terraform_state \
+    -target=module.run_shell_ssm_document \
+    -target=module.vpc_flow_logs \
+    -target=module.read_terraform_state \
```

Comment:
Fixing the order and `module.read_terraform_state` is duplicated.

```suggestion
    -target=module.email_sending_domain_certreadrole \
    -target=module.guacamole_certreadrole \
    -target=module.read_terraform_state \
    -target=module.run_shell_ssm_document \
    -target=module.vpc_flow_logs \
    -target=null_resource.break_association_with_default_route_table \
```